gf-guide/metrics.md

```diff
@@ -66,37 +78,30 @@ The Microsoft [OpenType specification](https://www.microsoft.com/typography/otsp
-
-This will force MS Applications to use the `Typo` values instead of the `Win` values for line spacing. By doing this, we can freely set the `Win` values to avoid clipping and control the line height with the `Typo` values. It has the added benefit of future line height compatibility. When a new script is added, we simply change the `Win` values to the new `yMin` and `yMax`, without needing to worry if the line height have changed. Note that the `Use_Typo_Metric` flag is also called `fsSelection bit 7 `(related to how it is set in the OS/2 table).
-
--   In Glyphs.app, set `Use_Typo_Metrics` custom parameter to `true` in the **Font** tab of **Font Info**.
--   In RoboFont, this is under **Font Info \> OpenType \> OS/2 Table \> fsSelection \> USE_TYPO_METRICS**.
--   In the OS/2 table: `<fsSelection value="00000000 10000000"/>`
-
-**6. If the family is being updated, the line height must visually match the previous release.**
-
-Some applications do not allow users to control the line height/leading of their fonts. Word processors and text editors are common culprits. It is essential their documents do not reflow.
-
-**7. Hhea and Typo metrics should be equal.**
+#### 7. Hhea and Typo metrics should be equal
 
 `Hhea` metrics are used in macOS, whilst Microsoft uses `Typo` when `Use_Typo_Metrics` is enabled. They should ideally be identical.
 
 This rule can be avoided if a font is being upgraded and previously had inconsistent values.
 
-**8. LineGap values must be 0.**
+## 8. LineGap values must be 0
 
 The `LineGap` value is a space added to the line height created by the union of the `(typo/hhea)Ascender` and `(typo/hhea)Descender`. It is handled differently according to the environment. This leading value will be added above the text line in most desktop apps. It will be shared above *and* under in web browsers, and ignored in Windows if `Use_Typo_Metrics` is disabled. For better linespacing consistency across platforms, `(typo/hhea)LineGap` values must be `0`.
 
-**9. Uppercases should be centered if the font’s primary script has uppercases letterforms such as Latin, Greek and Cyrillic.**
+#### 9. Uppercases should be centered if the font’s primary script has uppercases letterforms such as Latin, Greek and Cyrillic
 
 Web designers will thank you if you managed to have the same space above and under capitals: `typoAscender - CapsHeight = abs(typoDescender)`. It will make easier for them the setting of padding in buttons for example.
 
-**10. typo/hheaAscender value should be greater than Agrave's yMax when it makes sense.**
+#### 10. typo/hheaAscender value should leave open room for stacked diacritics.
+
+Following recent tests to confirm our VM policies, we now require that the typo/hheaAscender be equal to `A breve acute`. For families with multiple weights you must use the tallest `A breve acute` (in e.g., the Black master) as the reference point to guarantee uniform positioning across the entire font family.
```

Comment:
Change "VM policies" to "vertical metrics policies"?

Reply:
Done